server/conf/application.conf

```diff
@@ -603,3 +603,7 @@ civiform_applicant_enabled_languages = ${?CIVIFORM_APPLICANT_ENABLED_LANGUAGES}
 # Typically, this value is 1.
 num_trusted_proxies = 1
 num_trusted_proxies = ${?NUM_TRUSTED_PROXIES}
+
+# These are file types allowed to be uploaded by the file upload components
+file_upload_allowed_file_type_specifiers = "image/*,.pdf"
```

Comment:
question (non-blocking): Since the default is set here, do we need the fallback logic in the classes? I'm concerned about the different default places getting out of sync. Also, should you add the default to the dev and test confs?

Reply:
It's there only as a last resort in the event someone sets the config value in their deploy to nothing. Better to have the minimum "safe" types so it doesn't completely error out. I'm not too worried about the java classes getting out of sync since long term the non-northstar version will go away.

We don't need to add them to dev/test conf files. They all inherit it from application.conf.

Reply:
Gotcha. I'm not worried about the java classes getting out of sync, but the conf file getting out of sync with the list in the northstar class for example. Maybe add a comment in the java class to keep that list in sync with the one in the conf file?

Reply:
Sure. I added a note on the northstar class. Not going to bother with the other one.